### PR TITLE
Add Northern Virginia to GCE zones list

### PR DESCRIPTION
**Pull request by alimakki**
vet-northern-virginia-1200x825

### CODE REVIEW

File: playbooks/google.yml [hunk 1]
```diff
@@ -7,26 +7,29 @@
...
-      "16": "asia-northeast1-a"
-      "17": "asia-northeast1-b"
-      "18": "asia-northeast1-c"
-      "19": "asia-southeast1-a"
-      "20": "asia-southeast1-b"
+      "1":  "us-central1-a"
+      "2":  "us-central1-b"
+      "3":  "us-central1-c"
+      "4":  "us-central1-f"
+      "5":  "us-east1-b"
```
Comment (nickolasclarke) [inline playbooks/google.yml]: these all seem to be mapped backwards. "us-east1-b" = (Northern Virginia A) in the vars_prompt.

Comment (alimakki) [inline playbooks/google.yml]: Should be corrected now.
Thanks for catching that.

**Comment by nickolasclarke** [comment]
lgtm 👍

**Comment by cpu** [comment]
Thanks @alimakki

**Pull request merged**